Sources/BraintreeVisaCheckout/BTVisaCheckoutClient.swift

```diff
@@ -1,6 +1,90 @@
+import VisaCheckoutSDK
 import Foundation
 
+#if canImport(BraintreeCore)
+import BraintreeCore
+#endif
+
+// Creates a Visa Checkout client for processing Visa Checkout payments.
 public class BTVisaCheckoutClient {
 
-    // TODO: - Add implementation
+    private let apiClient: BTAPIClient
+
+    /// Creates a Visa Checkout client.
+    ///
+    /// - Parameters:
+    ///   - apiClient: An API client.
+    @objc(initWithAPIClient:)
+    public init(apiClient: BTAPIClient) {
+        self.apiClient = apiClient
+    }
+
+    /// Creates a Visa Checkout `ProfileBuilder` with the merchant API key, environment, and other properties to be used with Visa Checkout.
+    ///
+    /// - Parameters:
+    ///   - completion: A completion block that is invoked when the profile is created. If the profile creation succeeds,
+    ///     `Profile` will contain a profile and `error` will be `nil`.
+    ///     If it fails, `Profile` will be `nil` and `error` will describe the failure.
+    @objc public func createProfileBuilder(completion: @escaping (Profile?, Error?) -> Void) {
+        apiClient.fetchOrReturnRemoteConfiguration { configuration, error in
+            if let error {
+                /// TODO: Add failure analytics event
+                completion(nil, error)
+                return
+            }
+
+            guard let configuration, let json = configuration.json else {
```

Comment:
This is causing your cocoapods error since it's not being used. Can be removed.
```suggestion
            guard let configuration else {
```